src/XamlStyler.Console/XamlStyler.Console.csproj

```
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <PackageId>XamlStyler.Console</PackageId>
-    <PackageVersion>3.1.0.0</PackageVersion>
+    <PackageVersion>3.1.1.0</PackageVersion>
```

```suggestion
    <PackageVersion>3.1.0.0</PackageVersion>
```

Revert change of build number as it's owned by the pipeline? Though I'd suggest to release changing to .NET 6 target as at least a `3.2` vs. a `3.1.x`?

---

Actually, can you revert this change? Our build pipeline will overwrite versioning for release.